tests/main/interfaces-cpu-control/task.yaml

```diff
@@ -0,0 +1,56 @@
+summary: Ensure that the cpu-control interface works.
+
+details: |
+    The cpu-control interface allows setting CPU tunables
+
+prepare: |
+    # shellcheck source=tests/lib/snaps.sh
+    . "$TESTSLIB/snaps.sh"
+    install_local test-snapd-cpu-control
+
+restore: |
+    rm -f call.error
+
+execute: |
+    echo "The interface is not connected by default"
+    snap interfaces -i cpu-control |  MATCH -- '^- +test-snapd-cpu-control:cpu-control'
+
+    echo "When the interface is connected"
+    snap connect test-snapd-cpu-control:cpu-control
+
+    echo "Then the snap is able to read the online file for the cpu0"
+    # Write is not allowed so it is not tested, even when the interface allows that
+    if [ -f /sys/devices/system/cpu/cpu0/online ]; then
```

Comment:
Neither of those files exist on my system, which means the test does nothing here. Is there anything else more common that the interface offers access to, so that we can be sure this test isn't a NOOP?